aten/src/ATen/CMakeLists.txt

```diff
@@ -211,7 +211,7 @@ endif()
 
 if(AT_NNPACK_ENABLED)
   include_directories(${NNPACK_INCLUDE_DIRS})
-  list(APPEND ATen_CPU_DEPENDENCY_LIBS nnpack) # cpuinfo is added below
+  list(APPEND ATen_CPU_DEPENDENCY_LIBS nnpack nnpack_reference_layers) # cpuinfo is added below
```

Comment:
NNPACK does not support strided convolutions on batches (used during training,) hence we fallback to the slower reference implementation here.